src/components/wynncraft/item/MajorIds.tsx

```
@@ -0,0 +1,23 @@
+  majorIds: {
+    [key: string]: string;
+  };
+}
+
+const MajorIds: React.FC<MajorIdsProps> = ({ majorIds }) => {
+  if (!majorIds || Object.keys(majorIds).length === 0) return null;
+
+  return (
+    <div className="list-disc list-inside">
+      {Object.entries(majorIds).map(([key, value]) => (
+        <div className="text-sm" key={key} dangerouslySetInnerHTML={{ __html: value }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)